nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -46,7 +46,8 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
   auto node = shared_from_this();
 
   // Support for handling the topic-based goal pose from rviz
-  client_node_ = std::make_shared<rclcpp::Node>("bt_navigator_client_node");
+  client_node_ =
+      std::make_shared<rclcpp::Node>("bt_navigator_client_node", rclcpp::NodeOptions().use_global_arguments(false));
```

Comment:
Looks like the code is exceeding 100 columns here and will fail the cpplint tool. Please run ament_cpplint and ament_uncrustify. We're trying to get all code to pass cleanly.